bin/sequenceserver

```diff
@@ -245,6 +245,8 @@ MSG
           puts
           puts 'Installing NCBI BLAST+.'
           puts "RUBY_PLATFORM #{RUBY_PLATFORM}"
+          puts 'Archive will be extracted in ~/.sequenceserver directory.'
```

Comment:
@yeban It should work as well. Although, I'm not sure how to interpret it if the *installation* fails. Was it installed? We can be even more verbose I guess.